ontology/yaml/resources/states/states.yaml

```diff
@@ -70,6 +70,20 @@ NOSOURCE: "An operation mode having no source as priority"
 POSITIONA: "Automatic Transfer switch position 0"
 POSITIONB: "Automatic Transfer switch position 1"
 POSITIONC: "Automatic Transfer switch position 2"
+
+ALARM: "Device is not in normal operation"
+DIMMING: "Dimmer is operated by setting a constant dimming setpoint"
+CONSTANT_LIGHT_CONTROL: "Dimmer controls the room illuminance"
+SCENE_CONTROL: "Dimmer activates scene presets"
+NORTH: "Cardinal direction; opposite of south"
+SOUTH: "Cardinal direction; opposite of north"
+EAST: "Cardinal direction; opposite of west"
+WEST: "Cardinal direction; opposite of east"
+DAY_NIGHT: "Operation mode for Blinds"
+SCENE: "Operation mode for Blinds"
```

Comment:
Is there a better definition and state? I dont know what SCENE mode would even be, and the definition is teh same for all modes. These should be more descriptive.